sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -222,37 +227,21 @@ export interface ListSettingsOptions extends OptionalFields {
     labelFilter?: string;
 }
 
-// @public
-export interface ListSettingsSnapshotsOptions extends OperationOptions, OptionalFields {
-    acceptDateTime?: Date;
-}
-
 // @public
 export interface ListSnapshots extends OptionalSnapshotFields {
     nameFilter?: string;
     statusFilter?: SnapshotStatus[];
 }
 
 // @public
-export interface ListSnapshotsOptions extends OperationOptions, ListSnapshots {
+export interface ListSnapshotsOptions extends OperationOptions, ListSnapshots, OptionalSnapshotFields {
 }
 
 // @public
 export interface ListSnapshotsPage extends HttpResponseField<SyncTokenHeaderField>, PageSettings {
     items: Snapshot[];
 }
 
-// @public
-export interface OperationDetails {
```

Comment:
where did this go?

Reply:
I removed many exports because they were unused. I only found references for these in the generated layer.